### PR TITLE
Updated python SMAP tutorial

### PR DESCRIPTION
**Pull request by zschira**
Updated SMAP tutorial to use new servers/api

### CODE REVIEW

File: python/create-raster-from-smap-soil-moisture-data.ipynb [hunk 1]
```diff
@@ -6,22 +6,22 @@
    "source": [
     "---\n",
     "layout: single\n",
-    "title: 'Create rasters from HDF5 SMAP data in Python'\n",
+    "title: 'Create rasters from SMAP data in Python'\n",
     "date: 2016-06-28\n",
     "authors: [Matt Oakley]\n",
```
Comment (mbjoseph) [inline python/create-raster-from-smap-soil-moisture-data.ipynb]: Can you add your name here to the list of authors?

File: python/create-raster-from-smap-soil-moisture-data.ipynb [hunk 2]
```diff
@@ -151,186 +114,467 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Step 4: Creating a Raster Stack\n",
-    "\n",
-    "After running the above code, we have individual raster images (.tif files); the number of which depends on the amount of datasets you used (in this example we used two). Now that we have these individual rasters, we want to merge or \"stack\" them on top of each other. GDAL has a python script which accomplishes just this. The result of running this will be a new .tif file called out.tif which is the merging or 'stacking' of the 2 rasters we created in Step 2."
+    "Once the data has been extracted you can load the geotiff files using GDAL to create a raster."
```
Comment (mbjoseph) [inline python/create-raster-from-smap-soil-moisture-data.ipynb]: geotiff should be GeoTIFF for consistency

**Comment by zschira** [comment]
Made those changes

**Comment by mbjoseph** [comment]
Awesome, thanks @zschira!

**Pull request merged**